applications/FluidDynamicsApplication/custom_utilities/mass_conservation_utility.cpp

```diff
@@ -169,46 +248,38 @@ void MassConservationCheckProcess::ComputeVolumesAndInterface( double& positiveV
         Matrix shape_functions;
         GeometryType::ShapeFunctionsGradientsType shape_derivatives;
 
-        const auto rGeom = it_elem->GetGeometry();
+        auto r_geometry = it_elem->GetGeometry();
```

Comment:
```suggestion
        const auto& r_geometry = it_elem->GetGeometry();
```
Missing `&`. Besides I think it can be `const`.

Reply:
(this probably requires to change the arguments in the declaration of `IsGeometryCut`).